python/seldon_core/microservice.py

```diff
@@ -419,11 +425,18 @@ def rest_prediction_server():
     else:
         # Start production server
         def rest_prediction_server():
+            rest_timeout = DEFAULT_ANNOTATION_REST_TIMEOUT
+            if ANNOTATION_REST_TIMEOUT in annotations:
+                # Timeout is in miliseconds so convert as annotation is in miliseconds
+                rest_timeout = int(annotations[ANNOTATION_REST_TIMEOUT])/1000
```

Comment:
@axsaucedo isn't the annotation already in MS? And doesn't Gunicorn take the timeout in MS?

Reply:
Oh good shout - the comment is wrong, it basically is supposed to say that "gunicorn timeout is in seconds"

Reply:
Ok updated the comment 👍